intro-HPC/ch_easybuild.tex

```diff
@@ -0,0 +1,139 @@
+
+\section{When should I use Easybuild?}
+
+For general software installation requests, please see \autoref{sec:software-installation}. However,
+there might be reasons to install the software yourself:
+
+\begin{enumerate}
+    \item applying custom patches to the software that only you or your group are using
+    \item evaluating new software versions prior to requesting a central software installation
+    \item installing (very) old software versions that are no longer eligible for central installation (on new systems)
+\end{enumerate}
+
+\section{Configuring EasyBuild}
+
+Before you use EasyBuild, you need to configure it:
+
+\subsection{Path to sources}
+
+This is where EasyBuild can find software sources:
+
+\begin{prompt}
+%\shellcmd{export EASYBUILD\_SOURCEPATH=/apps/gent/source}%
+\end{prompt}
+
+\subsection{Build directory}
+
+This directory is where EasyBuild will build software in. To have good performance,
+this needs to be on a fast filesystem.
+
+\begin{prompt}
+%\shellcmd{export EASYBUILD\_BUILDPATH=\${TMPDIR:-/tmp/\$USER}}%
+\end{prompt}
+
+On cluster nodes, you can use the fast, in-memory \lstinline|/dev/shm/$USER| location
+as a build directory.
+
+\subsection{Software install location}
+
+This is where EasyBuild will install the software (and accompanying modules) to.
+
+For example, to let it use \lstinline|$VSC_DATA/easybuild|, use:
+
+
+% Don't put the whole command inside \shellcmd{} because it would show ugly white boxes
+
+\begin{prompt}
+%\shellcmd{export}% EASYBUILD_INSTALLPATH=$VSC_DATA/easybuild/$VSC_OS_LOCAL/$VSC_ARCH_LOCAL$VSC_ARCH_SUFFIX
+\end{prompt}
+
+Using the \lstinline|$VSC_ARCH| and \lstinline|$VSC_ARCH_SUFFIX| environment variables
+ensures that your install software to a location that is specific to the cluster you're
+building for.
+
+Make sure you \strong{do not build software on the login nodes}, since the loaded \lstinline|cluster|
+module determines the location of the installed software. Software built on the login
+nodes may not work on the cluster you want to use the software on.
+
+To share custom software installations with members of your VO, replace \lstinline|$VSC_DATA|
+with \lstinline|$VSC_DATA_VO| in the example above.
+
+\section{Using EasyBuild}
+
+Before using EasyBuild, you first need to load the \lstinline|EasyBuild| module. We don't specify
+a version here (this is an exception, for other modules we would) because newer versions
+might include important bugfixes.
+
+\begin{prompt}
+module load EasyBuild
+\end{prompt}
+
+\subsection{Installing supported software}
+
+EasyBuild provides a large collection of readily available software versions,
+combined with a particular toolchain version. Use the \lstinline|--search|
+(or \lstinline|-S|) functionality to see which different 'easyconfigs'
+(build recipes, see \url{http://easybuild.readthedocs.org/en/latest/Concepts_and_Terminology.html#easyconfig-files}) are available:
+
+\begin{prompt}
+%\shellcmd{eb -S OpenFOAM-2.2}%
```

Comment:
It's probably better to use a different example than `OpenFOAM` here, because building your own OpenFOAM is quite painful, so we shouldn't promote it...

Maybe use a fictional `example-1.2` search, and show a handful of results (`example-1.2.3-foss-\year{}a.eb`, etc.)?